timeline/tests/test_views.py

```diff
@@ -193,3 +193,33 @@ def test_render(self):
             self.assertEqual(
                 response.context['object_list'].first(), self.event_site
             )
+
+
+class TestAdminEventListView(TestViewsBase):
+    """Tests for the admin timeline list view"""
+
+    def setUp(self):
+        super().setUp()
+        self.event_site = self.timeline.add_event(
+            project=None,
+            app_name='projectroles',
+            user=self.user,
+            event_name='test_event',
+            description='description',
+            extra_data={'test_key': 'test_val'},
+        )
+
+    def test_render(self):
+        """Test rendering the admin list view"""
+        with self.login(self.user):
+            response = self.client.get(
+                reverse(
+                    'timeline:timeline_site_admin',
+                )
+            )
+            self.assertEqual(response.status_code, 200)
```

Comment:
One small hint: when we do a request with `with self.login()`, the only rows later which must be contained within the `with` block are ones for asserting response redirecting (as it does some stuff which requires the user to be logged in). If you want to avoid unnecessary indentation, you can just de-indent the rest of these rows :)